frontend/packages/operator-lifecycle-manager/src/status/csv-status.ts

```diff
@@ -31,21 +32,21 @@ export const subscriptionForCSV = (
 export const getCSVStatus = (
   csv: ClusterServiceVersionKind,
 ): { status: ClusterServiceVersionStatus; title: string } => {
-  const statusPhase = _.get(csv, 'status.phase', ClusterServiceVersionPhase.CSVPhaseUnknown);
+  const statusPhase = csv?.status?.phase ?? ClusterServiceVersionPhase.CSVPhaseUnknown;
   let status: ClusterServiceVersionStatus;
   if (pedingPhases.includes(statusPhase)) {
     status = ClusterServiceVersionStatus.Pending;
   } else {
     switch (statusPhase) {
       case ClusterServiceVersionPhase.CSVPhaseSucceeded:
-        status = ClusterServiceVersionStatus.OK;
+        status = i18n.t(`olm~${ClusterServiceVersionStatus.OK}`);
```

Comment:
I think it's less error-prone to do it this way as long as we always use this function to show the status. Then you don't risk missing a string or having the i18next-parser comments get out of sync with the enum value.

```suggestion
        status = i18n.t('olm~OK');
```